operator/src/main/java/com/github/streamshub/console/dependents/ConsoleDeployment.java

```diff
@@ -58,6 +64,11 @@ protected Deployment desired(Console primary, Context<Console> context) {
         String imageAPI = imagesSpec.map(Images::getApi).orElse(defaultAPIImage);
         String imageUI = imagesSpec.map(Images::getUi).orElse(defaultUIImage);
 
+        var envVars = new ArrayList<>(coalesce(primary.getSpec().getEnv(), Collections::emptyList));
```

Comment:
I don't understand why `coalesce` is called coalesce.  it seems to be `valueOrDefault()`

Reply:
This is my SQL history coming in :laughing: ... it's also named something similar in JavaScript [1]. Functionally it's `valueOrDefault`, but less verbose.

[1] https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing